src/components/RecentBlogPosts/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import posts from '@site/.docusaurus/docusaurus-plugin-content-blog/default/blog-posts-metadata.json';
+import RecentBlog from '../RecentBlogPost';
+
+const RecentBlogPosts = ({ blogs_count }) => {
+    const recentPosts = posts.slice(0, blogs_count);
+
+    if (recentPosts.length === 0) {
+        return <p>No recent blog posts available.</p>;
```

Comment:
I think it is better to return nothing in this case.

Reply:
will return null